src/components/dataset/overview/DatasetOverviewPanel.jsx

```diff
@@ -178,6 +211,78 @@ function DatasetOverviewPanel(props) {
             <Typography variant="h6">Cloud Platforms:</Typography>
             {renderCloudPlatforms(dataset)}
           </Grid>
+          <Grid item xs={4}>
+            <Typography variant="h6">Dataset Id:</Typography>
+            <Typography>{dataset.id}</Typography>
+          </Grid>
+          <Grid item xs={4}>
+            <Typography variant="h6">Default Billing Profile Id:</Typography>
+            {dataset.defaultProfileId}
+          </Grid>
+          {dataset.dataProject && (
+            <Grid item xs={4}>
+              <Typography variant="h6">Google Data Project:</Typography>
+              {dataset.dataProject}
+            </Grid>
+          )}
+          {dataset.ingestServiceAccount && (
+            <Grid item xs={4}>
+              <Typography variant="h6">Ingest Service Account:</Typography>
+              <Typography>{dataset.ingestServiceAccount}</Typography>
+            </Grid>
+          )}
+          <Grid item xs={4}>
+            <Typography variant="h6">Secure Monitoring Enabled?</Typography>
+            {dataset.secureMonitoringEnabled ? 'Yes' : 'No'}
+          </Grid>
+          <Grid item xs={4}>
+            <Typography variant="h6">Self Hosted?</Typography>
+            {dataset.selfHosted ? 'Yes' : 'No'}
+          </Grid>
+          <Grid item xs={4}>
+            <EditableFieldView
+              fieldValue={dataset.phsId}
+              fieldName="PHS Id"
+              canEdit={userRoles.includes(DatasetRoles.STEWARD)}
+              updateFieldValueFn={(text) => dispatch(patchDataset(dataset.id, { phsId: text }))}
+              useMarkdown={false}
+            />
+          </Grid>
+          {dataset.schema.relationships.length > 0 && (
+            <Grid item xs={8}>
+              <Accordion defaultExpanded={false}>
+                <AccordionSummary
+                  expandIcon={<ExpandMore className={classes.expandIcon} />}
+                  className={classes.header}
+                >
+                  Relationships ({dataset.schema.relationships.length ?? 0})
+                </AccordionSummary>
+                <AccordionDetails data-cy="relationship">
+                  {dataset.schema.relationships.length === 0 && (
+                    <Typography className={classes.noRelationships}>(None)</Typography>
+                  )}
+                  {dataset.schema.relationships.map((rel, i) => (
+                    <div key={rel.name}>
+                      <Typography variant="h6">{rel.name}</Typography>
+                      <Typography variant="h6">From</Typography>
+                      Table: {rel.from.table}
```

Comment:
For formatting and accessibility, would this be worth separating out the key and value into different elements?

Reply:
Oh, that would be great to make it more accessible! 
I'm not exactly sure what you mean by your suggestion. Can you elaborate?